test/views/ViewTestHelper.scala

```diff
@@ -37,6 +37,24 @@ trait ViewTestHelper extends SpecBase {
     view.title() mustBe s"${messages(titleMsgKey)} - ${messages("service.name")} - GOV.UK"
 
   def shouldContainBackLinkUrl(view: Document,
-                               url: String): Assertion =
-    view.html().contains(url) mustBe true
+                               url: String): Assertion = view.html().contains(url) mustBe true
+
+  def shouldContainTheElement(view: Document,
+                              id: Option[String] = None,
+                              classes: Option[String] = None): Assertion = {
+    val isElementIdPresent = id.fold(false)(id => view.select(s"#$id").size() >= 0)
```

Comment:
Here you are checking whether the element count is `>= 0`  (i.e., non-negative), which is always true. The check should be for `> 0`, as an element is considered present only when the count is greater than zero.

I think the current logic will incorrectly pass the assertion even if the element is not present, as `>= 0`  is true for any count.

Reply:
Done, condition has been updated to > 0